app/static/out_of_office/out_of_office.js

```diff
@@ -0,0 +1,16 @@
+function findselected() {
+    let deactivate = 0;
+    let result = document.querySelector('input[name="outOfOffice"]:checked').value;
+    if (result == deactivate) {
+        document.getElementById("start_date").setAttribute('disabled', true);
+        document.getElementById("end_date").setAttribute('disabled', true);
+        document.getElementById("start_time").setAttribute('disabled', true);
+        document.getElementById("end_time").setAttribute('disabled', true);
+    }
+    else {
+        document.getElementById("start_date").removeAttribute('disabled');
+        document.getElementById("end_date").removeAttribute('disabled');
+        document.getElementById("start_time").removeAttribute('disabled');
```

Comment:
same

Reply:
why that? It's separated elements that need to be disabled and enabled together :)